gui/src/sounds/sounds.tsx

```diff
@@ -0,0 +1,18 @@
+import { ResetType } from 'solarxr-protocol';
+
+export function playSoundForTriggered(resetType: ResetType) {
+  switch (resetType) {
+    case ResetType.Quick: {
+      new Audio('/sounds/single_beep.wav').play();
+      break;
+    }
+    case ResetType.Full: {
+      new Audio('/sounds/double_beep.wav').play();
+      break;
+    }
+    case ResetType.Mounting: {
+      new Audio('/sounds/triple_beep.wav').play();
+      break;
+    }
+  }
+}
```

Comment:
Could you put the audio elements in a const outside the function like
```js
const singleBeep = new Audio('/sounds/single_beep.wav')
// ...
singleBeep.play()
```

Audio elements shouldn't be played the moment they are created. This is mostly for browsers tho, in where there is an actual latency of buffering the audio.

Reply:
Do it anyways though, because we do have a browser version that people can use!

Reply:
> You should make this a draft, if it isn't finished yet :p.

I think I completed the PR ytd and it is ready for review.

Sorry for the notification.

Reply:
fixed!